lib/dor/workflow/client/status.rb

```diff
@@ -99,12 +102,14 @@ def simplified_status_code(display)
 
         def current_milestones
           current = []
-          # only get steps that are part of accessioning and part of the current version. That can mean they were archived with the current version
-          # number, or they might be active (no version number).
-          milestones.each do |m|
-            if STEPS.key?(m[:milestone]) && (m[:version].nil? || m[:version] == version)
-              current << m unless m[:milestone] == 'registered' && version.to_i > 1
-            end
+          # only get steps that are part of accessioning and part of the current version.
+          # That can mean they were archived with the current version number, or they might be active (no version number).
+          milestones.each do |mstone|
```

Comment:
yeah.  the real problem is that it probably should have been ":name" or ":label" instead of ":milestone" for the name of the thing.  It's not obvious that "milestone" is a hash and one of the keys is "milestone" which will be the name/label/type of the thing.